_inc/client/at-a-glance/index.jsx

```diff
@@ -34,11 +34,11 @@ import {
 import { isDevMode } from 'state/connection';
 import { getModuleOverride } from 'state/modules';
 
-const renderPairs = layout => layout.map( item => (
+const renderPairs = layout => layout.map( ( item, id ) => (
 	[
 		item.header,
-		chunk( item.cards, 2 ).map( ( [ left, right ] ) => (
-			<div className="jp-at-a-glance__item-grid">
+		chunk( item.cards, 2 ).map( ( [ left, right ], idx ) => (
+			<div className="jp-at-a-glance__item-grid" key={ 10 * id + idx }>
```

Comment:
Something like this might read better (those keys are technically indexes instead of ids) but up to you. 🙃 
```js
key={ `card-${ layoutIndex }-${ cardIndex }` }
```

Reply:
Thanks! Implemented in 177c2b0